server.js

```diff
@@ -0,0 +1,25 @@
+var express = require('express');
+
+var mongoose = require('mongoose');
+
+var rcCollectionRouter = require('./rcCollectionroutes');
+
+var rcRecordRouter = require('./rcRecordroutes');
+
+var app = express();
+
+var db = mongoose.connect('mongodb://localhost/reCollector');
+
+app.use(express.static('public'));
+
+app.use('/rcCollection', rcCollectionRouter);
+
+app.use('/rcRecord', rcRecordRouter)
+
+app.use('/serverModule', router);
```

Comment:
`router` is undefined - does this need to be required in, or is this line even necessary?